src/App.jsx

```diff
@@ -1,28 +1,91 @@
-import React, { useState } from 'react';
+import React, { useState, useRef, useEffect } from 'react';
 import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd';
+import EmojiPicker from 'emoji-picker-react';
 import './App.css';
 
 const initialColumns = {
   todo: {
     id: 'todo',
     title: 'To Do',
-    tasks: []
-  },
-  inProgress: {
-    id: 'inProgress',
-    title: 'In Progress',
-    tasks: []
-  },
-  done: {
-    id: 'done',
-    title: 'Done',
-    tasks: []
+    tasks: [],
+    order: 0,
+    emoji: '📝',
+    color: '#1a73e8'
   }
 };
 
 function App() {
   const [columns, setColumns] = useState(initialColumns);
   const [newTask, setNewTask] = useState('');
+  const [newColumnTitle, setNewColumnTitle] = useState('');
+  const [boardTitle, setBoardTitle] = useState('Kanban Board');
+  const [isEditingTitle, setIsEditingTitle] = useState(false);
+  const titleInputRef = useRef(null);
+  const [showEmojiPicker, setShowEmojiPicker] = useState(null);
+  const [showColorPicker, setShowColorPicker] = useState(null);
+  const emojiPickerRef = useRef(null);
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
const emojiPickerRef = useRef(null);
const colorPickerRef = useRef(null);

// Using separate refs
<div className="emoji-picker-container" ref={emojiPickerRef}>
<div className="color-picker-container" ref={colorPickerRef}>
```
The click-outside handler uses the same ref (emojiPickerRef) for both emoji picker and color picker containers. This could lead to unexpected behavior when both pickers are rendered. Consider using separate refs for each picker to ensure proper click-outside handling.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;